kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/activity/ConversationActivity.java

```diff
@@ -624,7 +627,7 @@ public void setToolbarTitleSubtitleColorFromSettings() {
     }
 
     @Override
-    public void onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) {
+    public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
```

Comment:
The crash can still happen with this annotation. Add a the null check in if-else for better handling.

Reply:
Added as suggested